src/contexts/yjs-context.tsx

```diff
@@ -33,91 +31,16 @@ export function YjsProvider({ children }: YjsProviderProps) {
 	// useBindValtioToYjs(medicationRegimensProxy, doc.getArray('medication-regimens'));
 	// useBindValtioToYjs(medicationsProxy, doc.getArray('medications'));
 
-	useBindValtioToYjs(diaperChanges, doc.getArray('diaper-changes-dec'));
-	useBindValtioToYjs(events, doc.getArray('events-dec'));
-	useBindValtioToYjs(feedingSessions, doc.getArray('feeding-sessions-dec'));
-	useBindValtioToYjs(
-		growthMeasurements,
-		doc.getArray('growth-measurments-dec'),
-	);
-	useBindValtioToYjs(feedingInProgress, doc.getMap('feeding-in-progress-dec'));
+	useBindValtioToYjs(diaperChanges, doc.getArray('diaper-changes'));
```

Comment:
Here and in other places you are changing the name of the collection. This is not allowed, it must stay the same. This is the name of our database table.

Reply:
You have not reverted the change. All collections need to end with `-dec`!